src/main/java/com/example/enterpriseapplication/EnterpriseApplication.java

```diff
@@ -15,92 +15,6 @@
 public class EnterpriseApplication {
 
 	public static void main(String[] args) {
-
 		SpringApplication.run(EnterpriseApplication.class, args);
-		class Recipe {
-			private String name;
-			private String description;
-			private List<String> ingredients;
-
-			// Getters and setters
-			// ...
-
-			@Override
-			public String toString() {
-				return "Recipe{" +
-						"name='" + name + '\'' +
-						", description='" + description + '\'' +
-						", ingredients=" + ingredients +
-						'}';
-			}
-		}
```

Comment:
Completely removed this class as a Recipe dto already exists, thus reducing technical debt. A string override can alos be completed in the dto itself.

https://martinfowler.com/bliki/TechnicalDebt.html